tests/components/utility_meter/test_init.py

```diff
@@ -46,7 +46,7 @@ async def test_restore_state(hass):
     )
 
     assert await async_setup_component(hass, DOMAIN, config)
-    assert await async_setup_component(hass, SENSOR_DOMAIN, config)
+    assert await async_setup_component(hass, Platform.SENSOR, config)
```

Comment:
This is questionable since `async_setup_comonent` takes any domain and not just platform domains as parameter.